rct229/rulesets/ashrae9012019/section5/section5rule10.py

```diff
@@ -134,5 +134,9 @@ def manual_check_required(self, context, calc_vals=None, data=None):
             def rule_check(self, context, calc_vals=None, data=None):
                 floor_u_factor = calc_vals["floor_u_factor"]
                 target_u_factor = calc_vals["target_u_factor"]
+                return floor_u_factor == target_u_factor
 
+            def is_tolerance_fail(self, context, calc_vals=None, data=None):
+                floor_u_factor = calc_vals["floor_u_factor"]
+                target_u_factor = calc_vals["target_u_factor"]
                 return std_le(val=floor_u_factor, std_val=target_u_factor)
```

Comment:
Use `std_equal` instead of `std_le` for now.
The `std_le` later can use for AHJ conservative check.